### ISSUE TITLE
Add live reload for slides

### ISSUE THREAD

**Issue by vinayak-mehta** [vinayak-mehta/present#24]
It would be nice to do this with `pyinotify` or `watchdog`; or maybe just pressing 'r'.

**Comment by vinayak-mehta** [comment]
With a `present --watch` option?

**Comment by mrshu** [comment]
It seems like `r` may be a nice option here -- it's also what [patat](https://github.com/jaspervdj/patat) does.

**Comment by vinayak-mehta** [comment]
:+1:

**Comment by mrshu** [comment]
All right, I take it back -- it seems like `r` is already used for resetting `codio`: https://github.com/vinayak-mehta/present/commit/b8e87dc8a4e522f514f0529728d4a8a8064143ab

**Comment by jorgenbs** [comment]
> All right, I take it back -- it seems like `r` is already used for resetting `codio`: [b8e87dc](https://github.com/vinayak-mehta/present/commit/b8e87dc8a4e522f514f0529728d4a8a8064143ab)

That only resets a codio-effect if present, as I understand it.

**Comment by vinayak-mehta** [comment]
Yes, currently `r` only resets a codio if its present on a slide. We can probably change the codio reset keybinding to something else, and have `r` do a hard reload for the current / all slides.
